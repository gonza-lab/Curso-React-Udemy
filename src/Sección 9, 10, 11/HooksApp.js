import React from 'react';
import { TodoApp } from './components/08-useReducer/TodoApp';


export const HooksApp = () => {

  return (
    <>
      <TodoApp />
    </>
  );
};
