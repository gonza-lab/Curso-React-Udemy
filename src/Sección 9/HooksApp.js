import React from 'react';
import { Padre } from './components/07-tarea-memo/Padre';


export const HooksApp = () => {

  return (
    <>
      <Padre />
    </>
  );
};
