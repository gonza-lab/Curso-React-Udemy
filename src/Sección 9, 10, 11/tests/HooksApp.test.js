import React from 'react'
import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';

describe('Pruebas en HooksApp', () => {
  test('Debe mostrar el componente correctamente', () => {
    expect(shallow(<HooksApp />)).toMatchSnapshot();
  });
});
