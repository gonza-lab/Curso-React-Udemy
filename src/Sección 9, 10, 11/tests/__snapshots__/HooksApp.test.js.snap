// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HooksApp Debe mostrar el componente correctamente 1`] = `
<Fragment>
  <TodoApp />
</Fragment>
`;
